src/PhpSpreadsheet/Calculation/Engineering/BesselJ.php

```diff
@@ -167,7 +167,7 @@ private static function besselj2b(float $ax, int $ord, float $x)
             if ($jsum === true) {
                 $sum += $bj;
             }
-            $jsum = !$jsum;
```

Comment:
Why does scrutinizer force us to add the overhead of a method call?

A more efficient alternative might be:
```php
$jsum = $jsum !== true;
```
though Scrutinixer might complain about that as well

Reply:
I will try that later today as well. I suspect it also won't work, but agree that would be a nicer solution.

Reply:
See PR #3123, which I will delete in a day or two. YES ... this did eliminate Scrutinizer's objection. I will update the appropriate places in this PR accordingly, tonight or tomorrow.

Reply:
It's also perhaps easier to read than `$x = !$x` where it's easy to miss the `!` when just glancing at the code, and doesn't have the overhead of a function call

Reply:
Changed in latest push.